keel-sql/src/main/resources/db/changelog/20200522-standardize-timestamps.yml

```diff
@@ -1,39 +1,153 @@
+            constraints:
+            - nullable: false
+    - sql:
+        sql: |
+          update diff_fingerprint set first_detection_time = from_unixtime(first_detection_time_old / 1000);
+    - dropColumn:
+        tableName: diff_fingerprint
+        columnName: first_detection_time_old
+- changeSet:
+    id: standardize-timestamps-environment_artifact_pin
+    author: fletch
+    changes:
+    - renameColumn:
+        tableName: environment_artifact_pin
+        oldColumnName: pinned_at
+        newColumnName: pinned_at_old
+        columnDataType: bigint(13)
+    - addColumn:
+        tableName: environment_artifact_pin
+        columns:
+        - column:
+            name: pinned_at
+            type: timestamp(3)
+            constraints:
+            - nullable: false
+    - sql:
+        sql: |
+          update environment_artifact_pin set pinned_at = from_unixtime(pinned_at_old / 1000);
+    - dropColumn:
+        tableName: environment_artifact_pin
+        columnName: pinned_at_old
+- changeSet:
+    id: standardize-timestamps-environment_artifact_queued_approval
+    author: fletch
+    changes:
+    - renameColumn:
+        tableName: environment_artifact_queued_approval
+        oldColumnName: queued_at
+        newColumnName: queued_at_old
+        columnDataType: bigint(13)
+    - addColumn:
+        tableName: environment_artifact_queued_approval
+        columns:
+        - column:
+            name: queued_at
+            type: timestamp(3)
+            constraints:
+            - nullable: false
+    - sql:
+        sql: |
+          update environment_artifact_queued_approval set queued_at = from_unixtime(queued_at_old / 1000);
+    - dropColumn:
+        tableName: environment_artifact_queued_approval
+        columnName: queued_at_old
+- changeSet:
+    id: standardize-timestamps-task_tracking
+    author: fletch
+    changes:
+    - renameColumn:
+        tableName: task_tracking
+        oldColumnName: timestamp
+        newColumnName: timestamp_old
+        columnDataType: bigint(13)
+    - addColumn:
+        tableName: task_tracking
+        columns:
+        - column:
+            name: timestamp
+            type: timestamp(3)
+            constraints:
+            - nullable: false
+    - sql:
+        sql: |
+          update task_tracking set timestamp = from_unixtime(timestamp_old / 1000);
+    - dropColumn:
+        tableName: task_tracking
+        columnName: timestamp_old
+- changeSet:
+    id: standardize-timestamps-unhappy_veto
+    author: fletch
+    changes:
+    - renameColumn:
+        tableName: unhappy_veto
+        oldColumnName: recheck_time
+        newColumnName: recheck_time_old
+        columnDataType: bigint(13)
+    - dropNotNullConstraint:
+        tableName: unhappy_veto
+        columnName: recheck_time_old
+        columnDataType: bigint(13)
+    - sql:
+        sql: |
+          update unhappy_veto set recheck_time_old = null where recheck_time_old = 9223372036854775807;
```

Comment:
With `unhappy_veto` we no longer want to be using this `Long.MAX_VALUE` to represent "never re-check" since it can't be converted to a timestamp. Instead that will now be indicated by `null`.